src/main/java/io/axoniq/demo/university/faculty/FacultyModuleConfiguration.java

```diff
@@ -12,13 +15,25 @@
 public class FacultyModuleConfiguration {
 
     public static EventSourcingConfigurer configure(EventSourcingConfigurer configurer) {
+        // shared infrastructure
+        configurer = NotificationServiceConfiguration.configure(configurer);
+
+        // Write side
         configurer = CreateCourseConfiguration.configure(configurer);
         configurer = CreateCoursePlainConfiguration.configure(configurer);
         configurer = RenameCourseConfiguration.configure(configurer);
         configurer = ChangeCourseCapacityConfiguration.configure(configurer);
         configurer = SubscribeStudentConfiguration.configure(configurer);
         configurer = SubscribeStudentMultiEntityConfiguration.configure(configurer);
         configurer = UnsubscribeStudentConfiguration.configure(configurer);
+
+        // Read side
+        // TBD
```

Comment:
For when? :angel: 
I do think at least a single read side example before we release AF5RC1 is important to have.

Reply:
OK, yes - it's an option. I've added it here:
https://github.com/AxonFramework/AxonFramework/issues/3659